manifests/services.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: minecraft-minecraft
+  labels:
+    app: minecraft-minecraft
+    chart: "minecraft-3.1.3"
+    release: "minecraft"
+    heritage: "Helm"
+  annotations:
+    {}
+spec:
+  clusterIP: 172.30.48.134
+  clusterIPs:
+  - 172.30.48.134
+  externalTrafficPolicy: Cluster
+  ports:
+  - nodePort: 31904
+    port: 25565
+    protocol: TCP
+    targetPort: 25565
+  selector:
+    app: minecraft-minecraft
+  sessionAffinity: None
+  type: NodePort
```

Comment:
Will this cause any issues?

Reply:
Ordinary user can't use `NodePort` in Openshift..

https://docs.openshift.com/container-platform/4.7/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport.html#nw-using-nodeport_configuring-ingress-cluster-traffic-nodeport

I don't think it will cause issues per se, but you must ensure the IP is available to the cluster and static...

Reply:
Ok, let's just assume this :)